payments/voucher_manager.go

```diff
@@ -51,9 +51,8 @@ func (vm *VoucherManager) Remove(channelId types.Destination) error {
 // Pay will deduct amount from balance and add it to paid, returning a signed voucher for the
 // total amount paid.
 func (vm *VoucherManager) Pay(channelId types.Destination, amount *big.Int, pk []byte) (Voucher, error) {
-	vInfo, ok := vm.store.GetVoucherInfo(channelId)
-
-	if !ok {
+	vInfo, err := vm.store.GetVoucherInfo(channelId)
+	if err != nil {
 		return Voucher{}, fmt.Errorf("channel not found")
```

Comment:
We should probably [wrap](https://go.dev/blog/go1.13-errors) the error returned by GetVoucherInfo when creating the new error.